gnome3.aisleriot: Build failure

## Issue description
`gnome3.aisleriot` fails to build --- it looks like something is going wrong when autoconf tries to find guile.

### Steps to reproduce
`nix-build '<nixpkgs>' -A gnome3.aisleriot`

## Technical details

* System: `NixOS 17.09pre104936.b0d07aa894 (Hummingbird)`
* Nix version: `nix-env (Nix) 1.11.8`
* Nixpkgs version: `17.09pre105622.f0fac3b578`

Build Output:
````
[ahmedtd@basis:~/project/nixpkgs]$ nix-build '<nixpkgs>' -A gnome3.aisleriot
these derivations will be built:
  /nix/store/fmr37kcjpl6zcxqvzbs8y4jqdqrx2bhr-aisleriot-3.22.0.drv
building path(s) ‘/nix/store/yplqhk1igzq50hjb6hl0b0mrd0lyrf5v-aisleriot-3.22.0’
unpacking sources
unpacking source archive /nix/store/p5qfp9j6is95y4jbzkknij9p6q5pk5ij-aisleriot-3.22.0.tar.xz
source root is aisleriot-3.22.0
setting SOURCE_DATE_EPOCH to timestamp 1474212823 of file aisleriot-3.22.0/ChangeLog
patching sources
configuring
fixing libtool script ./ltmain.sh
configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/yplqhk1igzq50hjb6hl0b0mrd0lyrf5v-aisleriot-3.22.0 --with-card-theme-formats=svg 
checking for a BSD-compatible install... /nix/store/ar26y4qyr0cwkp75d4mppj474rzfmk6h-coreutils-8.27/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /nix/store/ar26y4qyr0cwkp75d4mppj474rzfmk6h-coreutils-8.27/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking whether to enable maintainer-specific portions of Makefiles... yes
checking whether to enable debugging... no
checking for pkg-config... /nix/store/4xic3cvzhx94nsq5dbnvrymsgnlggdk5-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.15... yes
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... none
checking how to run the C preprocessor... gcc -E
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... none
checking whether ln -s works... yes
checking for a sed that does not truncate output... /nix/store/9n2ddr7yw374cc35g08zwhzbcdrz2d3r-gnused-4.4/bin/sed
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /nix/store/9n2ddr7yw374cc35g08zwhzbcdrz2d3r-gnused-4.4/bin/sed
checking for grep that handles long lines and -e... /nix/store/w9rnix7bzpv18152nk97g2d38kin1j3b-gnugrep-3.0/bin/grep
checking for egrep... /nix/store/w9rnix7bzpv18152nk97g2d38kin1j3b-gnugrep-3.0/bin/grep -E
checking for fgrep... /nix/store/w9rnix7bzpv18152nk97g2d38kin1j3b-gnugrep-3.0/bin/grep -F
checking for ld used by gcc... /nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld
checking if the linker (/nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /nix/store/8yb0smqwyf3da339b5kjhw8vwlslrdp0-binutils-2.28/bin/nm -B
checking the name lister (/nix/store/8yb0smqwyf3da339b5kjhw8vwlslrdp0-binutils-2.28/bin/nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /nix/store/8yb0smqwyf3da339b5kjhw8vwlslrdp0-binutils-2.28/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /nix/store/ar26y4qyr0cwkp75d4mppj474rzfmk6h-coreutils-8.27/bin/dd
checking how to truncate binary pipes... /nix/store/ar26y4qyr0cwkp75d4mppj474rzfmk6h-coreutils-8.27/bin/dd bs=4096 count=1
./configure: line 7664: /usr/bin/file: No such file or directory
checking for mt... no
checking if : is a manifest tool... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld
checking if the linker (/nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld) is GNU ld... yes
checking whether the g++ linker (/nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... no
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/nix/store/dqfikdyxdrykwcbg1wp6nd725y0c7va4-gcc-wrapper-5.4.0/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for which platform to build... gnome
checking which gtk+ version to compile against... 3.0
checking for some Win32 platform... no
checking for native Win32... no
checking for requested card theme formats... svg
checking distribution... unknown
checking which card theme to use by default... gnomangelo_bitmap.svgz
checking which card theme format to use by default... svg
checking whether extra debugging UI in Aisleriot is requested... no
checking whether to enable sound support... yes
checking for pkg-config... (cached) /nix/store/4xic3cvzhx94nsq5dbnvrymsgnlggdk5-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.16... yes
Using config source xml:merged:/nix/store/fx6rxhmcig9dqspz8gams55id3jxwj9j-gconf-3.2.6/etc/gconf/gconf.xml.defaults for schema installation
Using $(sysconfdir)/gconf/schemas as install directory for schema files
checking for gconftool-2... /nix/store/fx6rxhmcig9dqspz8gams55id3jxwj9j-gconf-3.2.6/bin/gconftool-2
checking for glib-genmarshal... /nix/store/vc1ipmdkvnkx5p99vadzvlan6c6cyhbh-glib-2.50.3-dev/bin/glib-genmarshal
checking for glib-compile-resources... /nix/store/vc1ipmdkvnkx5p99vadzvlan6c6cyhbh-glib-2.50.3-dev/bin/glib-compile-resources
checking for gzip... /nix/store/fcpdji0ycbsnlw03z7m4b062z03ia91z-gzip-1.8/bin/gzip
checking for xmllint... /nix/store/hb44wqfj9k51g8i453acxrlzm2q4vf1l-libxml2-2.9.4-bin/bin/xmllint
checking for desktop-file-validate... /nix/store/c7spngrw5klqp9lkm540k929n0n474ds-desktop-file-utils-0.22/bin/desktop-file-validate
checking whether to enable binary relocation support... no
checking if gcc supports -Wall flag... yes
checking if gcc supports -Wextra flag... yes
checking if gcc supports -Wformat-nonliteral flag... yes
checking if gcc supports -Werror=format-security flag... yes
checking if gcc supports -Wsign-compare flag... yes
checking if gcc supports -Werror=implicit-function-declaration flag... yes
checking if gcc supports -Wpointer-arith flag... yes
checking if gcc supports -Wwrite-strings flag... yes
checking if gcc supports -Wstrict-prototypes flag... yes
checking if gcc supports -Waggregate-return flag... yes
checking if gcc supports -Wcast-align flag... yes
checking if gcc supports -Wimplicit flag... yes
checking if gcc supports -Wuninitialized flag... yes
checking if gcc supports -Wmissing-prototypes flag... yes
checking if gcc supports -Wmissing-declarations flag... yes
checking if gcc supports -Wnested-externs flag... yes
checking if gcc supports -Wpacked flag... yes
checking if gcc supports -Wmissing-format-attribute flag... yes
checking if gcc supports -Wshadow flag... yes
checking if gcc supports -Wlogical-op flag... yes
checking if gcc supports -Wstrict-aliasing=2 flag... yes
checking if gcc supports -Winit-self flag... yes
checking if gcc supports -Wunsafe-loop-optimizations flag... yes
checking if gcc supports -Wdeclaration-after-statement flag... yes
checking if gcc supports -Wold-style-definition flag... yes
checking if gcc supports -Wno-missing-field-initializers flag... yes
checking if gcc supports -Wno-unused-parameter flag... yes
checking if gcc supports -Wempty-body flag... yes
checking if gcc supports -Wformat-signedness flag... yes
checking if gcc supports -fno-common flag... yes
checking if gcc supports -fno-strict-aliasing flag... yes
checking if gcc supports -Wno-switch-enum flag... yes
checking locale.h usability... yes
checking locale.h presence... yes
checking for locale.h... yes
checking for LC_MESSAGES... yes
checking libintl.h usability... yes
checking libintl.h presence... yes
checking for libintl.h... yes
checking for ngettext in libc... yes
checking for dgettext in libc... yes
checking for bind_textdomain_codeset... yes
checking for msgfmt... /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/msgfmt
checking for dcgettext... yes
checking if msgfmt accepts -c... yes
checking for gmsgfmt... /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/msgfmt
checking for xgettext... /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/xgettext
checking whether NLS is requested... yes
checking for intltool >= 0.35.0... 0.51.0 found
checking for intltool-update... /nix/store/c8hw7jhg5dxpwdgfj703cvk0mmxyypdq-intltool-0.51.0/bin/intltool-update
checking for intltool-merge... /nix/store/c8hw7jhg5dxpwdgfj703cvk0mmxyypdq-intltool-0.51.0/bin/intltool-merge
checking for intltool-extract... /nix/store/c8hw7jhg5dxpwdgfj703cvk0mmxyypdq-intltool-0.51.0/bin/intltool-extract
checking for xgettext... (cached) /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/xgettext
checking for msgmerge... /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/msgmerge
checking for msgfmt... (cached) /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/msgfmt
checking for gmsgfmt... (cached) /nix/store/m91iwf035rbswpw76nkbic0dyyr7vdc4-gettext-0.19.8/bin/msgfmt
checking for perl... /nix/store/qxq74ic7ivdb3sx1x8bpn8sbmk63v5si-perl-5.22.2/bin/perl
checking for perl >= 5.8.1... 5.22.2
checking for XML::Parser... ok
checking which guile version to use... 2.0
checking for guile... /nix/store/zfd3ivd81hfycxs0c6nk5j0hrp3gggw2-guile-2.2.0/bin/guile
checking for AISLERIOT... no
configure: error: Package requirements ( gtk+-3.0 >= 3.4.0 libcanberra-gtk3 >= 0.26 gconf-2.0 >= 2.0 librsvg-2.0 >= 2.32.0 cairo >= 1.10.0 guile-2.0 >= 2.0.0) were not met:

No package 'guile-2.0' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables AISLERIOT_CFLAGS
and AISLERIOT_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
builder for ‘/nix/store/fmr37kcjpl6zcxqvzbs8y4jqdqrx2bhr-aisleriot-3.22.0.drv’ failed with exit code 1
error: build of ‘/nix/store/fmr37kcjpl6zcxqvzbs8y4jqdqrx2bhr-aisleriot-3.22.0.drv’ failed

````

--- a/pkgs/desktops/gnome-3/3.22/games/aisleriot/default.nix
+++ b/pkgs/desktops/gnome-3/3.22/games/aisleriot/default.nix
@@ -1,6 +1,6 @@
 { stdenv, fetchurl, pkgconfig, gnome3, intltool, itstool, gtk3
 , wrapGAppsHook, gconf, librsvg, libxml2, desktop_file_utils
-, guile, libcanberra_gtk3 }:
+, guile_2_0, libcanberra_gtk3 }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
@@ -8,7 +8,7 @@ stdenv.mkDerivation rec {
   configureFlags = [ "--with-card-theme-formats=svg" ];
 
   buildInputs = [ pkgconfig intltool itstool gtk3 wrapGAppsHook gconf
-                  librsvg libxml2 desktop_file_utils guile libcanberra_gtk3 ];
+                  librsvg libxml2 desktop_file_utils guile_2_0 libcanberra_gtk3 ];
 
   meta = with stdenv.lib; {
     homepage = https://wiki.gnome.org/Apps/Aisleriot;

